lookml/orders.view.lkml

```diff
@@ -70,51 +95,254 @@ view: orders {
       year
     ]
     sql: ${TABLE}.first_order_date ;;
+    group_label: "Order Dates"
   }
 
-  # ------------------------------------------------ Other
+  dimension_group: previous_completed_order {
+    description: "This is the customer's previous completed order timestamp"
+    type: time
+    timeframes: [
+      raw,
+      time,
+      date,
+      week,
+      month,
+      quarter,
+      year
+    ]
+    sql: ${TABLE}.previous_completed_order_date ;;
+    group_label: "Order Dates"
+  }
 
-  dimension: order_number {
+
+#------------------------------------------ ORDER ATTRIBUTES
+
+  dimension: order_seq_number {
+    hidden: yes
+    type: string
+    sql: ${TABLE}.order_seq_number ;;
+  }
+
+  dimension: order_year {
+    description: "This truncates the order created date to the year"
+    type: number
+    sql: ${TABLE}.order_year ;;
+    group_label: "Order Attributes"
+  }
+
+  dimension: order_month {
+    description: "This truncates the order created date to the month"
+    type: number
+    sql: ${TABLE}.order_month ;;
+    group_label: "Order Attributes"
+  }
+
+  dimension: order_day_of_month {
+    description: "This truncates the order created date to the day"
+    type: string
+    sql: ${TABLE}.order_day_of_month ;;
+  }
+
+  dimension: completed_order_number {
     type: number
+    sql: ${TABLE}.completed_order_number ;;
+    group_label: "Order Attributes"
+  }
+
+  dimension: is_completed {
+    description: "This returns '1' if order is completed (financial_status in ('paid', 'partially_paid', 'partially_refunded')), '0' if no"
```

Comment:
update this description- this is very specific to shopify. it should say this returns 1 if the parameters for a completed order have been fulfilled